generated/nidcpower/session.py

```diff
@@ -4151,6 +4151,599 @@ def export_attribute_configuration_file(self, file_path):
         errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=False)
         return
 
+    @ivi_synchronized
+    def create_advanced_sequence(self, sequence_name, sequence, set_as_active_sequence=True):
+        '''create_advanced_sequence
+
+        Test.
+
+        Note: Test.
```

Comment:
As you mention, documentation is still outstanding

Reply:
> I don't want to put in the effort to fix it in case we end up doing something different.

Sounds like documentation isn't fixed.  I'm unresolving this to help us, the reviewers, track this being resolved.